client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -81,5 +81,21 @@ describe( 'Site Migration Flow', () => {
 				state: { siteSlug: 'example.wordpress.com' },
 			} );
 		} );
+
+		it( 'redirects from upgrade-plan to bundleTransfer step after selecting a free trial', async () => {
```

Comment:
```suggestion
		it( 'redirects from upgrade-plan to bundleTransfer step after selecting a free trial', () => {
```

We don't need async here.
I probably also lefts some unnecessary async 🤔  on previous tests.